pkg/cmd/pr/edit/edit.go

```diff
@@ -182,6 +188,7 @@ func updatePullRequest(client *api.Client, repo ghrepo.Interface, id string, edi
 		PullRequestID: id,
 		Title:         ghString(editable.TitleValue()),
 		Body:          ghString(editable.BodyValue()),
+		BaseRefName:   ghString(editable.BaseValue()),
```

Comment:
Could we avoid passing the `BaseRefName` parameter unless we're sure that the value has been edited; i.e. that the `--base` flag has been passed?

This is to avoid race conditions.